lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1866,6 +1886,22 @@ def api_client
       @client
     end
 
+    def use_secure_channel
+      @logging_api_url.start_with?('https')
```

Comment:
Shouldn't this be `starts_with?('https://')`?

Reply:
Changed.